src/session/content/event_row.rs

```diff
@@ -91,5 +91,11 @@ impl EventRow {
 
     pub(crate) fn set_label(&self, label: &str) {
         self.imp().label.set_markup(label);
+
+        // Make sure event row won't occupy too much horizontal space
+        let label = self.imp().label.to_owned();
+        label.set_justify(gtk::Justification::Center);
+        label.set_wrap(true);
+        label.set_wrap_mode(pango::WrapMode::Char);
```

Comment:
You should set these properties inside the [ui file](https://github.com/melix99/telegrand/blob/main/data/resources/ui/content-event-row.ui) directly.

Reply:
right